grommet-leaflet-core/src/themes/base.js

```diff
@@ -0,0 +1,67 @@
+const markerKinds = {
+  default: {
```

Comment:
In Grommet, we tend to define all of these properties at the object's root. Here we are placing under a `default` property. Do we want to follow the existing Grommet pattern, or introduce this?

Most of the impacts are implementation details, however the theme interface becomes an API we are to which we are committing.

Reply:
It's good to consider further given it's an interface we are committing to.

In Button kind in grommet we took the approach of explicitly defining `default`. We also took that approach in some of the initial explorations with vanilla-extract.

I feel like being more explicit about the "default" is somewhat helpful, but I also am open to flattening.

I will leave as `default` in this PR but we can continue to simmer on pros/cons.